ui/components/app/nfts-items/nfts-items.js

```diff
@@ -182,13 +183,14 @@ export default function NftsItems({
                   className="nfts-items__item-wrapper"
                 >
                   <NftItem
-                    src={nftImage}
+                    src={image ? nftImage : ''}
```

Comment:
what is image vs nftImage?

Reply:
image is the open sea link for the image, nftImage gives the link based on ipfs, we are using getAssetUrl to get the value on line 173 `const nftImage = getAssetImageURL(imageOriginal, ipfsGateway);`